Makefile

```diff
@@ -55,7 +55,7 @@ build-wws-cross-%: install-cross
 
 .PHONY: build-spin
 build-spin:
-	cargo build --release --manifest-path=containerd-shim-spin-v1/Cargo.toml
+	rustup target add wasm32-unknown-unknown && cargo build --release --manifest-path=containerd-shim-spin-v1/Cargo.toml
```

Comment:
I propose to add new step called `install-rust-targets` that adds both wasm32-unknown-unknown and wasm32-wasi targets, and make it a prerequisite for `build-spin` and `build-slight`.

Reply:
Thanks for your advice. I modified a few places:
1. Makefile added install-rust-targets
2. build.yaml and ci.yaml use install-rust-targets